build.assets/Dockerfile

```diff
@@ -251,11 +251,11 @@ COPY --from=libfido2 /usr/local/lib/pkgconfig/ /usr/local/lib/pkgconfig/
 COPY --from=libfido2 \
     /usr/local/lib/libcbor.a \
     /usr/local/lib/libfido2.a \
-    /usr/local/lib/libfido2.so.1.11.0 \
+    /usr/local/lib/libfido2.so.1.12.0 \
```

Comment:
Why are we copying dynamic libraries at the end if we're not using them? AFAIK we only use the static one.

Reply:
IIRC we use the dynamic lib for `go test`. It can also be handy for manual fiddling in the containers.